src/App.tsx

```diff
@@ -25,6 +26,10 @@ export const App = () => {
                     <Route path="/" element={<MainPage />} />
                     <Route path="/pcs" element={<Products />} />
                     <Route path="/shopping-cart" element={<ShoppingCart />} />
+                    <Route
+                        path="/add-new-product"
```

Comment:
Zastanawiam sie jaka jest konwencja nazewnicza endpointów dla frontu do wyświetlania stron. Bo wiadomo API używa rzeczowników i metod HTTP, a tutaj można nazywać ścieżki bardziej opisowo?

Reply:
Nie znalazłem żadnych jako takich standardów nazewnictwa. Wzorowałem się na adresie z euro: 

`https://www.euro.com.pl/komputery-stacjonarne-pc/optimus-pc-optimus-ga520t-cr1-r5-16-1tb-6600-w11.bhtml`

- `komputery-stacjonarne-pc` - kategoria
- `optimus-pc-optimus-ga520t-cr1-r5-16-1tb-6600-w11` - jakiś identyfikator, ja użyłem `id`, aby było prościej

To co wyświetla się w URL nie ma żadnego związku z endpointami na backendzie, więc możemy nazywać jak chcemy.

Jak chcesz to możemy nazywać endpointy na froncie i backendzie tak samo.

Przy URL z tego co wiem ważne jest aby link był unikatowy, bo możemy zrobić tak, że nawet jak wybierzemy konkretny produkt to link nie musi zawierać informacji pozwalającej na zidentyfikowanie produktu, ale taki link nie będzie pozwalał na powrót do tego produktu.

Pytałem znajomych frontendowców i każdy robi tak, aby URL na froncie przedstawił taką `ścieżkę` jak było klikane.